python/lsst/meas/algorithms/readTextCatalogTask.py

```diff
@@ -54,6 +54,12 @@ class ReadTextCatalogConfig(pexConfig.Config):
         doc=("Format of files to read, from the astropy.table I/O list here:"
              "http://docs.astropy.org/en/stable/io/unified.html#built-in-table-readers-writers")
     )
+    fill_values = pexConfig.ListField(
+        dtype=str,
+        default=[],
+        doc=("A list giving [<match_string>, '0', <optional column name>], which is used to mask"
+             " the given values in the input file.")
```

Comment:
Please reference the astropy doc that gives more detail on this, since this is really an astropy setting.

Why can't we use `NaN` for the fill value? The Mask Tables look like they support a per-column fill_value.

Reply:
If you put `Nan` in, the columns that should be integers get converted to floats. For astropy, you can do per-column `fill_value`s, but then we need to use `pexConfig.ConfigDictField`, and will have to know all the column names and set them individually in the task configuration. It's a little annoying, but maybe worth it. I'll try implementing it.